### ISSUE TITLE
PSNR

### ISSUE THREAD

**Issue by liushuangmax** [DeokyunKim/Progressive-Face-Super-Resolution#10]
hello,Why don't you compare the performance PSNR  of 2x and 4x?

**Comment by DeokyunKim** [comment]
The purpose of the research is x8 upscaling. Therefore, we did not compare and PSNR of 2x and 4x with baseline researches.

**Comment by liushuangmax** [comment]
@DeokyunKim But if x2,x4 is not good, will x8 be good?

**Comment by DeokyunKim** [comment]
The results of the last iteration in each step were experimentally good. Probably, if the results of x2 and x4  are not good, then x8 is also not good.

**Comment by liushuangmax** [comment]
@DeokyunKim hello,I would like to know if you have used your dataset to train and test other people's networks when comparing them to other people's methods, or if you have used your testset to test other people's networks directly

**Comment by yangyingni** [comment]
> @DeokyunKim hello,I would like to know if you have used your dataset to train and test other people's networks when comparing them to other people's methods, or if you have used your testset to test other people's networks directly

Thank you very much.And the input size shoule be 16*16 or bicubiced 128*128?

**Comment by DeokyunKim** [comment]
> @DeokyunKim hello,I would like to know if you have used your dataset to train and test other people's networks when comparing them to other people's methods, or if you have used your test set to test other people's networks directly

We utilize pre-trained FSRNet and FSRGAN with the different datasets to compare with our methods, and we train the VSDR with the same data set with our trained model. We mentioned that in the paper.

**Comment by yangyingni** [comment]
I trained other people's network with my datasets.
 



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "notifications"<notifications@github.com&gt;; 
发送时间: 2020年1月9日(星期四) 晚上9:34
收件人: "DeokyunKim/Progressive-Face-Super-Resolution"<Progressive-Face-Super-Resolution@noreply.github.com&gt;; 
抄送: "泱渶"<2676426668@qq.com&gt;; "Comment"<comment@noreply.github.com&gt;; 
主题: Re: [DeokyunKim/Progressive-Face-Super-Resolution] PSNR (#10)



 
@DeokyunKim hello,I would like to know if you have used your dataset to train and test other people's networks when comparing them to other people's methods, or if you have used your test set to test other people's networks directly
  
We utilize pre-trained FSRNet and FSRGAN with the different datasets to compare with our methods, and we train the VSDR with the same data set with our trained model. We mentioned that in the paper.
